examples/async_tasks/async_compute.rs

```diff
@@ -104,10 +104,10 @@ fn spawn_tasks(mut commands: Commands) {
     }
 }
 
-/// This system queries for entities that have our Task<Transform> component. It polls the
-/// tasks to see if they're complete. If the task is complete it takes the result, adds a
-/// new [`Mesh3d`] and [`MeshMaterial3d`] to the entity using the result from the task's work, and
-/// removes the task component from the entity.
+/// This system queries for entities that have our Task<CommandQueue> component. It polls the
+/// tasks to see if they're complete. If the task is complete it takes the result, and appends
+/// all the commands that were queued to the command object, so they will actually run.
+/// No application specific code is needed here, so this system could be in a separate plugin.
```

Comment:
The component used is `ComputeTransform`, `Tasks` are not components.

This also means that the last line you added is wrong

Reply:
yup the first line was already present, but it's wrong. `Task` can't be a component

Reply:
It does not say Task _**is**_ a component, it is referring to the component as the 'Task Component', i.e. the Component **_with_** the Task. Like if i call someone 'The motorcycle guy', I am not saying the guy *is* a motorcycle i am identifying him by the fact he *has* a motorcycle.  If i say 'The money box' i am referring to the box with the money in it, i am not saying the money is a box. That is I think a fairly natural reading of the sentence in English, and was how i genuinely interpreted it on my first reading of it, and although i didn't write it, it seems perfectly fine to me.

Reply:
I have re-worded 'Task component' -> 'Component containing task' to make it more explicit, and also re-worded last line.